Python/crossfire/crossfire/occurrences.py

```diff
@@ -1,18 +1,39 @@
 from queue import Empty, Queue
 from urllib.parse import urlencode
 
+from crossfire.errors import CrossfireError
+
+TYPE_OCCURRENCES = {"all", "withVictim", "withoutVictim"}
+
+
+class UnknownTypeOccurrenceError(CrossfireError):
+    def __init__(self, type_occurrence):
+        message = ("""Unknown type_occurrence""",)
+        f"`{type_occurrence}`. Valid formats are: {', '.join(TYPE_OCCURRENCES)}"
```

Comment:
Você testou isso manualmente? Essa sintaxe está muito estranha.

```python
      message = ("""Unknown type_occurrence""",)
```

Pq `message` precisa ser uma tupla com um único elemento?
Pq esse único elemento, uma _string_, precisa ser definida com `"""` ao invés de simplesmente uma aspas?

Essa outra linha só cria uma string em memória, sem atribuir a nenhuma variável… qual o objetivo?

```python
        f"`{type_occurrence}`. Valid formats are: {', '.join(TYPE_OCCURRENCES)}"
```

Reply:
Ouch... afobação minha,
A mensagem de erro estava com muitos caracteres em uma mesma linha, fazendo o `ruff` reclamar.
Por isso decidi quebrar um mais de uma linha. 
Achei que para isso eu precisaria usar doc-string, foi pesquisar mas não terminei minha tarefa e deixei incompleta.
Pesquisei agora, sobre como quebrar uma mensagem em varias linhas e usando f-tring e vi a possibilidade de usar parenteses para concatenarlas, inclusive previsto na PEP8.
Logo, corrigiria para:
```python
class UnknownTypeOccurrenceError(CrossfireError):
    def __init__(self, type_occurrence):
        message = (
            "Unknown type_occurrence"
            f"`{type_occurrence}`. Valid formats are: {', '.join(TYPE_OCCURRENCES)}"
        )
        super().__init__(message)
```
Concorda?

Reply:
Saquei. Por aí, só acho mais fácil de ler se balancear melhor o tamanho de cada linha, por exemplo:

```suggestion
        message = (
            f"Unknown type_occurrence `{type_occurrence}`. "
            f"Valid formats are: {', '.join(TYPE_OCCURRENCES)}"
        )
```